.github/workflows/build_test_deploy.yml

```
@@ -263,44 +263,65 @@ jobs:
 
       - uses: docker/login-action@v3
+        if: >
+          github.actor != 'dependabot[bot]' &&
+          github.event.pull_request.head.repo.full_name == github.repository
         with:
           username: ${{ secrets.DOCKER_USER }}
           password: ${{ secrets.DOCKER_PASS }}
 
       - name: Push webknossos image
-        if: ${{ ! startsWith(github.ref, 'refs/tags/') }}
+        if: >
```

actionlint (1.7.7)

if: condition "${{ ! startsWith(github.ref, 'refs/tags/') }} && github.actor != 'dependabot[bot]' && github.event.pull_request.head.repo.full_name == github.repository\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)